{{cookiecutter.github_repository}}/compose/dev/django/start

```diff
@@ -6,5 +6,8 @@ set -o nounset
 
 
 python /app/manage.py collectstatic --noinput
-# /usr/local/bin/gunicorn asgi --bind 0.0.0.0:5000 --chdir=/app -k uvicorn.workers.UvicornWorker
+{%- if cookiecutter.add_asgi.lower() == "y" %}
+/usr/local/bin/gunicorn asgi --bind 0.0.0.0:5000 --chdir=/app -k uvicorn.workers.UvicornWorker
+{%- else %}
 /usr/local/bin/gunicorn wsgi --bind 0.0.0.0:5000 --chdir=/app --access-logfile - --error-logfile -
+{%- endif %}
```

Comment:
Let's use port 8000 for the sake of consistency.

Reply:
Check if you can use `guincorn` directly without the whole path.

Reply:
Done and tested